.github/workflows/macos_unit_tests.yaml

```
@@ -0,0 +1,54 @@
+name: macos tests
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+jobs:
+  build:
+
+    runs-on: macos-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Python packages
+      run: |
+          pip install --upgrade pip
+          pip install --upgrade six
+          pip install --upgrade coverage
+          pip install --upgrade setuptools
+          pip install --upgrade codecov coverage==4.5.4
+          pip install --upgrade flake8
+          pip install --upgrade pep8-naming
+    - name: Setup Homebrew packages
+      run: |
+        brew update
+        brew upgrade
+        brew install gcc gnupg2 dash kcov
```

Maybe try: https://github.com/kelaberetiv/TagUI/issues/86
```suggestion
        brew install gcc gnupg2 dash kcov openssl
```

---

It didn't work here 6e22abe65c64dc4ca42e1161c38e0b345b2a91c4 but there are also weird interactions with the order in which `pip` and `brew` are called so I'll give it a try again.